### DISCUSSION TITLE
Can't reliably search for user activity in discussions without bending over backwards

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#86618] [Discussions]
### Select Topic Area

Product Feedback

### Body

- Consider [1] https://github.com/community/community/discussions/86501?notification_referrer_id=NT_kwDOACBWLLI5MDMzMDk3NzU2OjIxMTkyMTI#discussioncomment-8096087 which feels like spam, but may or may not be.
- [2] https://github.com/search?q=Vahid8478&type=discussions does not find it, even though I know it's a thing [1].
- [3] https://github.com/orgs/community/discussions?discussions_q=commenter%3AVahid8478 does turn it up, but [4] https://github.com/orgs/community/discussions?discussions_q=Vahid8478 does not

This doesn't work either: [5] https://github.com/search?q=repo%3Acommunity%2Fcommunity+commenter%3AVahid8478&type=discussions, even though it should really be the same as [3].

Context: I regularly report users for spamming, and if they make PRs / file issues, it's pretty easy to do this, but if they do weird things to discussions, it's really hard to figure out if they're an active spammer throughout github or it was a one-off.

---

- Consider [07510480632](https://github.com/07510480632), amongst the many _possibly_ spammy commenters on https://github.com/community/community/discussions/84131, searching for them doesn't show anything: https://github.com/search?q=07510480632&type=discussions, but it turns out via https://github.com/orgs/community/discussions?discussions_q=commenter%3A07510480632 that they've in fact been marked as a spammer elsewhere: https://github.com/community/community/discussions/77255#discussioncomment-7733832

**Comment by DailyMegaNews** [comment]
It can be tricky tracking patterns across discussions like this, especially when some results only appear through specific queries. Using a reliable [webp image text reader](https://quickimagetotext.com/webp-to-text) can sometimes help extract hidden details from shared screenshots or images of posts.

**Comment by DailyMegaNews** [comment]
It seems like tracking patterns across discussions can get tricky, especially when trying to distinguish a one-off case from consistent spam, much like choosing the right [digital marketing services company](https://ambsandigital.com/) requires careful analysis of subtle signals.
